crates/db_views/src/post_view.rs

```diff
@@ -391,8 +391,12 @@ impl<'a> PostQuery<'a> {
     }
 
     query = match self.sort.unwrap_or(SortType::Hot) {
-      SortType::Active => query.then_order_by(post_aggregates::hot_rank_active.desc()),
-      SortType::Hot => query.then_order_by(post_aggregates::hot_rank.desc()),
+      SortType::Active => query
+        .filter(post_aggregates::hot_rank_active.gt(1))
```

Comment:
Hot ranks fade to zero after a few days anyway.

Reply:
Better mention such things in comments inside the code.

Reply:
this filters by >1 not >0 as would seem to be suggested by the comment. not sure if intentional